src/parsec_service/build_run.md

```diff
@@ -74,14 +76,20 @@ the name `protobuf-compiler`:
 sudo apt install protobuf-compiler
 ```````
 
+To use `bindgen` and generate the Rust to C wrappers, `libclang` (version at least 3.9) is needed:
+
+```````
+sudo apt install llvm-dev libclang-dev clang
```

Comment:
This is for Debian-based distros - do we want to specify the command for other distributions/for MacOS too?

Reply:
I think ideally yes, but I guess most people know how to translate the command to the ones of their own distro, assuming the package names do not change.